tasks/sendBTC.ts

```diff
@@ -0,0 +1,148 @@
+import confirm from "@inquirer/confirm";
+import * as bitcoin from "bitcoinjs-lib";
+import * as dotenv from "dotenv";
+import ECPairFactory from "ecpair";
+import { task } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import * as ecc from "tiny-secp256k1";
+
+dotenv.config();
+
+const TESTNET = bitcoin.networks.testnet;
+
+const API = "https://blockstream.info/testnet/api";
```

Comment:
same comment here on if this api/testnet should be extracted into a config/package

Reply:
Added tBTC to `@zetachain/networks`: https://github.com/zeta-chain/toolkit/pull/10/commits/11be332dd7ef2c06ef73a4b011bff118c62d4834

As for `TESTNET`, these values are published as part of the `bitcoinjs-lib` package, so I don't think we need to publish it ourselves.